README.markdown

```diff
@@ -242,57 +254,86 @@ override func tableView(_ tableView: UITableView, numberOfRowsInSection section:
 ```
 ### Minimal Imports
 
-Keep imports minimal. For example, don't import `UIKit` when importing `Foundation` will suffice.
+Import only the modules a source file requires. For example, don't import `UIKit` when importing `Foundation` will suffice. Likewise, don't import `Foundation` if you must import `UIKit`.
 
-## Spacing
+**Preferred**:
+```
+import UIKit
+var view: UIView
+var deviceModels: [String]
+```
+
+**Preferred**:
+```
+import Foundation
+var deviceModels: [String]
+```
 
-* Indent using 4 spaces rather than tabs to conserve space and help prevent line wrapping. Check xcode settings at the end.
+**Not Preferred**:
+```
+import UIKit
+import Foundation
+var view: UIView
+var deviceModels: [String]
+```
+
+**Not Preferred**:
+```
+import UIKit
+var deviceModels: [String]
+```
 
+## Spacing
+
+* Indent using **4 spaces** rather than tabs to conserve space and help prevent line wrapping. Check xcode settings at the end.
```

Comment:
I'm a spaces guy but I do remember a discussion about accessibility of _spaces vs tabs_. More details can be found [here](https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/). What you think?

Reply:
In our case, even with Xcode editing settings, does it still differ using spaces and tabs?

Reply:
> In our case, even with Xcode editing settings, does it still differ using spaces and tabs?
yes, we can still configure Xcode to use tabs instead of spaces

I also prefer using spaces like we currently do, and given so far we haven't had anyone "complaining" that spaces aren't accessible for them, I would say we leave it as is until this becomes an actual problem. Also, this doesn't seem to have raised much discussion anyways 😆 

Nonetheless, these are just my 5¢ so feel free to continue the discussion.